README.md

```diff
@@ -72,6 +72,22 @@ end
 
 ## Configuration
 
+#### Singed assertions and Signed Response
+
+By default SAML Assertion will be signed with an algorithm which defined to `config.algorithm`. Because SAML assertions contain secure information used for authentication such as NameID.
+
+Signing SAML Response is optional, but some security perspective SP services might require Response message itself must be signed.
+For that, you can enable it with `config.signed_message` option. [More about SAML spec](https://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf#page=68)
+
+#### Signing algorithm
+
+Following algorithms you can set in your response signing algorithm
+:sha1 - RSA-SHA1 default value but not recommended to production environment
```

Comment:
why keeping `sha1` default and simply not removing it ?

any reason not to use `sha512` as default ?

Reply:
Sha1 is used for this gem signature feature. I don't want to start to change it when I reuse it.

Reply:
ok